docs/interpreting-experiments/confidence-intervals.md

```diff
@@ -1,34 +1,266 @@
 
-$\rho = \frac{10000}{\log( \log ( e \alpha^{-2} ) )-2\log{ \alpha }}$.
+In this section, we explain the strengths and weaknesses of each method to give you a sense of when they are most and least appropriate, given your experimentation constraints and goals. For the statistical details, including lots of Greek letters, see the [Statistical nitty-gritty](#statistical-nitty-gritty).
+
+
+:::caution
+
+All the above methods for estimating the lift rely on the [Central Limit Theorem](https://en.wikipedia.org/wiki/Central_limit_theorem) (CLT), which, for our purposes, states that, regardless of how a given metric is distributed among subjects, the *mean* of that metric has a [normal distribution](https://en.wikipedia.org/wiki/Normal_distribution) as the number of subjects gets large. The CLT only holds, however, when both control and treatment variants have enough subjects in them, so we do not display confidence intervals if either variant has less than 30 subjects.
+
+:::
+
+### Fixed-sample analysis
+
+A fixed-sample analysis is the most basic way to summarize the results from an experiment. First, you determine the minimum lift you care about (the [minimum detectable effect](../planning-experiments/minimum_detectable_effects.md), or MDE) for each metric you're going to observe in the experiment. Then, based on those MDEs, you do a power analysis to understand how many samples you need to be able to reliably detect an effect (for example, using Eppo's [Sample Size Calculator](../planning-experiments/index.md)). Then, you start your experiment, wait until you have the predetermined sample size, and *only then* do your analysis (and make a decision).
+
+The analysis itself consists of measuring the mean and variance of the lift you observed and then using those to conduct a [Null Hypothesis Significance Test](https://en.wikipedia.org/wiki/Statistical_hypothesis_testing#The_testing_process). In our case, the null hypothesis is that the treatment and control variants are identical, that is, that you are performing an A/A test; if it is unlikely[^3], if the null hypothesis *were* true, that we would observe a lift between the two variants that is as large as what was measured during the experiment, then we *reject* the hypothesis that both variants are identical and call the lift *statistically significant*.
```

Comment:
Here again I think it makes more sense to focus on confidence intervals, and just mention the relation to hypothesis testing in passing